ui/ui-expenses/src/screens/prices/needRefund/needRefund.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './needRefund.styles'
+
+const NeedRefund = ({ checked, onChange }) => (
+  <label htmlFor="needRefund" className={styles.price}>
+    Se faire rembourser ?
```

Comment:
With the question mark here we expect a yes or no answer. Since there is only one checkbox, I suggest to remove the "?"